src/tools/wpa/ETWTraceEvent.cs

```diff
@@ -7,9 +7,13 @@
 using Microsoft.Diagnostics.Tracing;
 using Microsoft.Performance.SDK.Extensibility;
 
-namespace QuicEventDataSource
+namespace MsQuicEtw
```

Comment:
Don't include ETW in the name. That's not going to be correct when we add LTTng support.

Reply:
I don't want it to just be MsQuic, because thats not what it is. `MsQuicTracing`?